rest/handler.go

```diff
@@ -336,12 +342,17 @@ func (h requestHandler) sendResponse(w http.ResponseWriter, ctx RequestContext)
 func sendResponse(w http.ResponseWriter, r response, serializer ResponseSerializer) {
 	status := r.Status
 	contentType := serializer.ContentType()
-	response, err := serializer.Serialize(r.Payload)
-	if err != nil {
-		log.Printf("Response serialization failed: %s", err)
-		status = http.StatusInternalServerError
-		contentType = "text/plain"
-		response = []byte(err.Error())
+
+	var response []byte
+	var err error
```

Comment:
I didn't want this here, but couldn't find a way around it. Leaving it out and doing `:=` in the `if` loop shadows `response`.